metaseq/models/transformer_decoder.py

```diff
@@ -422,6 +422,7 @@ def extract_features(
                 incremental_state=incremental_state,
                 self_attn_mask=self_attn_mask,
                 self_attn_padding_mask=self_attn_padding_mask,
+                recompute_fc1=(idx < getattr(self.args, "recompute_fc1_num_layers", 0)),
```

Comment:
Is this.. the first N layers by design? Sorry, don't have much context here.

Reply:
will add comment about it